Realm/Realm/Sync/ClientResyncMode.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ////////////////////////////////////////////////////////////////////////////
+// //
+// // Copyright 2021 Realm Inc.
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License")
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// // http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+// //
+// ////////////////////////////////////////////////////////////////////////////
+
+namespace Realms.Sync
+{
+        internal enum ClientResyncMode : byte
+        {
+            // Fire a client reset error
+            Manual = 0,
+
+            // Discard local changes, without disrupting accessors or closing the Realm
```

Comment:
We don't need these comments. Also, since DiscardLocal is the default, should it be 0?

Reply:
These map directly to [core values](https://github.com/realm/realm-core/blob/76661758162132e9af4245bc6842980e4e0fdd1c/src/realm/sync/config.hpp#L95). Doing so avoided the need for a native mirror in our wrappers that need to be converted to the core ones.

Reply:
I see, kind of weird they chose 0 for manual, but let's keep it aligned then.